server/models/postgis/project.py

```diff
@@ -826,6 +826,18 @@ def get_project_title(self, preferred_locale):
         )
         return project_info.name
 
+    @staticmethod
+    def get_project_total_contributions(project_id: int) -> int:
+
+        project_contributors_count = (
+            TaskHistory.query.with_entities(TaskHistory.user_id)
+            .filter(TaskHistory.project_id == project_id)
```

Comment:
I think we should exclude from the results the `COMMENT` actions.

Reply:
Agreed, good catch - implementing the change